README.md

```diff
@@ -1,471 +1,47 @@
-  types it will be shown as string-array, int-array...
-  
-In this configuration screen you can also choose a deserializer. Make sure to **chose our custom deserializer** when 
-you 
-are using a schema loaded from a file. (*net.coru.kloadgen.serializer.AvroDeserializer*)
+> *¿Tenemos logo de KLoadGen?*
 
-![Value Schema File Deserialization Configuration](/Value_Schema_File_Deserialization_Config.png)
+[Summary](#) · [Getting started](#) · [Usage](#) · [Support](#) · [Special thanks](#) 
 
-### Key Schema Deserialization Configuration
+## 📜 Summary
 
-Similar to Value Schema Deserialization Configuration, but focus to configure Key Schema deserialization. Whatever 
-settings selected here will be used to deserialize message Keys.
+KLoadGen is a Kafka load generator plugin for JMeter designed to work with AVRO and JSON schemas. It allows to send Kafka messages with a structure defined as an AVRO schema or a JSON schema. It connects to the Scheme Registry server, retrieves the subject to send, and generates a random message every time.
 
-### Key Schema File Deserialization Configuration
+### Documentation
 
-Similar to Value Schema File Deserialization Configuration, but focus to configure Key Schema deserialization. Whatever
-settings selected here will be used to deserialize message Keys.
+You can learn more details on our [wiki](https://github.com/corunet/kloadgen/wiki). 
 
-## Example Test Plan
 
-[Here](/Example-Test-Plan.jmx) you can find an example of a JMeter Test Plan using the elements defined in this plugin. This test plan will only inject messages in a Kafka Cluster. **Before** execute it you should add your Schema Registry to retrieve the Schema in order to extract the Entity structure. In a future we will include support to read AVRO files.
+### Technology
 
-## StandAlone execution
+KLoadGen is a plugin based on Java that needs a JMeter machine running JRE  8 or higher.
 
-This plugin also support an StandAlone execution. Only requires a JMeter installation in order to read some configuration files:
-* jmeter.properties
-* saveservice.properties
-* upgrade.properties
+## 🚀 Getting Started
 
-### Building
+Check our [Getting Started](https://github.com/corunet/kloadgen/wiki#getting-started) section for details on:
 
-Build process required specify the standalone profile
+- KLoadGen components
+- Plugin requirements
+- Installation process
 
-```bash
-$ mvn -P standalone clean install
-```
-### Execution
+## 🧑🏻‍💻 Usage
 
-Execution in standalone mode is quite easy :
+### Configuration
 
-```bash
-$ java -jar target/kloadgen-standalone-1.5.1.jar  -h ../JMeter/apache-jmeter-5.2\ 2 -l ../logs/results.log -t ../Example-Test-Plan.jmx -r ../logs
-```
+See the sections [Producer setup](https://github.com/corunet/kloadgen/wiki#producer-setup) and [Consumer setup](https://github.com/corunet/kloadgen/wiki#consumer-setup) on our wiki for details on how to set up this plugin.
```

Comment:
Please, explain briefly to users why they should "see the sections...". I think if you redirect to these two different links, the reason for going to one or the other will be different too. Explain it in a few words.